src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Could not add new actor: " + actor + " to DB");
```

Comment:
put also exception details from the Exception class: 
```suggestion
            throw new DataProcessingException("Could not add new actor: " + actor + " to DB", e);
```
please fix here an in similar cases in other `dao` classes.